src/vm/dwreport.cpp

```diff
@@ -2426,9 +2440,11 @@ FaultReportResult DoFaultReportWorker(      // Was Watson attempted, successful?
     pWatsonSharedMemory->bfmsoctdsLetRun = offerFlags;
 
     {
+        PathString wzModuleFileName;
         DWORD dwRet = WszGetModuleFileName(NULL,
-	                                       pWatsonSharedMemory->wzModuleFileName,
-	                                       NumItems(pWatsonSharedMemory->wzModuleFileName));
+	                                       wzModuleFileName);
+        wcsncpy_s(pWatsonSharedMemory->wzModuleFileName, NumItems(pWatsonSharedMemory->wzModuleFileName), wzModuleFileName, _TRUNCATE);
```

Comment:
Thanks for the correction Jeremy. Sounds like there is still no scenario here because of the specific API, but if I have misunderstood and we need answer on the original question do let me know.